package.json

```diff
@@ -29,7 +29,8 @@
     "@types/qs": "^6.9.6",
     "axios": "^0.21.1",
     "joi": "^17.4.0",
-    "qs": "^6.10.1"
+    "qs": "^6.10.1",
+    "rimraf": "^3.0.2"
```

Comment:
Can you put rimraf in the devDependencies? No need to have it in the implementing repo's (platform-client for example) I think?

Reply:
Updated